docs/blog/2023-07-15-incremental.md

```diff
@@ -0,0 +1,226 @@
+---
+slug: Changes
+title: Computing with changes
+authors: [mbudiu]
```

Comment:
Do you want your full name instead of email id?

Reply:
This is how this renders in the blog:
![image](https://github.com/feldera/dbsp/assets/19397042/f643e0fe-0c7b-4057-bc40-f80d3a48acbf)